source/ui.js

```diff
@@ -1,60 +1,28 @@
 'use strict';
 const inquirer = require('inquirer');
 const chalk = require('chalk');
-const githubUrlFromGit = require('github-url-from-git');
 const isScoped = require('is-scoped');
-const util = require('./util');
 const git = require('./git-util');
 const {prereleaseTags} = require('./npm/util');
 const version = require('./version');
 const prettyVersionDiff = require('./pretty-version-diff');
-
-const printCommitLog = async repoUrl => {
-	const latest = await git.latestTagOrFirstCommit();
-	const log = await git.commitLogFromRevision(latest);
-
-	if (!log) {
-		return {
-			hasCommits: false,
-			releaseNotes: () => {}
-		};
-	}
-
-	const commits = log.split('\n')
-		.map(commit => {
-			const splitIndex = commit.lastIndexOf(' ');
-			return {
-				message: commit.slice(0, splitIndex),
-				id: commit.slice(splitIndex + 1)
-			};
-		});
-
-	const history = commits.map(commit => {
-		const commitMessage = util.linkifyIssues(repoUrl, commit.message);
-		const commitId = util.linkifyCommit(repoUrl, commit.id);
-		return `- ${commitMessage}  ${commitId}`;
-	}).join('\n');
-
-	const releaseNotes = nextTag => commits.map(commit =>
-		`- ${commit.message}  ${commit.id}`
-	).join('\n') + `\n\n${repoUrl}/compare/${latest}...${nextTag}`;
-
-	const commitRange = util.linkifyCommitRange(repoUrl, `${latest}...master`);
-
-	console.log(`${chalk.bold('Commits:')}\n${history}\n\n${chalk.bold('Commit Range:')}\n${commitRange}\n`);
-
-	return {
-		hasCommits: true,
-		releaseNotes
-	};
+const getCommits = require('./get-commits');
+const printCommitLog = require('./print-commit-log');
+
+const generateReleaseNotes = (commits, repoUrl, latest, nextTag) => {
```

Comment:
Updated the default release notes format to match the one in "auto" mode. Let me know if you want to revert back to the original one without headings.

Reply:
The new format looks good 👌🏻